docs/translations/hu/scripting/functions/SetVehicleAngularVelocity.md/SetVehicleAngularVelocity.md

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
the file path is wrong, it seems you've accidentally created a folder with the intended file name